cmd/ttn-lw-cli/commands/end_devices.go

```diff
@@ -173,10 +171,6 @@ var (
 		Short:             "List available frequency plans for end devices",
 		PersistentPreRunE: preRun(),
 		RunE: func(cmd *cobra.Command, args []string) error {
-			if !config.GatewayServerEnabled {
-				return errGatewayServerDisabled
-			}
```

Comment:
Ah this is an old bug that I forgot to file an issue for.

For end devices, the authoritive source of frequency plans is the NS, not GS (like for gateways). So do check here if the NS is not enabled and error if so.